### PR TITLE
adjusted newlines and spaces; updated DFG links

### PR DESCRIPTION
**Pull request by Zack-83**
Cosmetics:
* removed unnecessary newlines, doubled significant ones
* removed extra spaces, replaced indenting spaces with tabs
* set thin unbreakable spaces before semicolon in French

### CODE REVIEW

File: rdmorganiser/questions/rdmo.xml [hunk 1]
```diff
@@ -3711,8 +3614,7 @@ In caso affermativo, si prega di descrivere succintamente le convenzioni per i n
...
-			
-Weitere Informationen zur Funktionsweise, Verwendung und den verschiedenen Arten von PIDs findet man z. B. im &lt;a href=&quot;https://www.dpconline.org/handbook/technical-solutions-and-tools/persistent-identifiers&quot; target=&quot;_blank&quot;&gt; Handbook der Digital Preservation Coalition&lt;/a&gt; oder in den Informationsmaterialien des &lt;a href=&quot;http://www.ands.org.au/guides#identify&quot; target=&quot;_blank&quot;&gt; Australian National Data Service (ANDS)&lt;/a&gt; (bitte scrollen Sie zu dem Abschnitt &quot;Identifying data and researchers&quot;).</help>
+ Weitere Informationen zur Funktionsweise, Verwendung und den verschiedenen Arten von PIDs findet man z. B. im &lt;a href=&quot;https://www.dpconline.org/handbook/technical-solutions-and-tools/persistent-identifiers&quot; target=&quot;_blank&quot;&gt; Handbook der Digital Preservation Coalition&lt;/a&gt; oder in den Informationsmaterialien des &lt;a href=&quot;http://www.ands.org.au/guides#identify&quot; target=&quot;_blank&quot;&gt; Australian National Data Service (ANDS)&lt;/a&gt; (bitte scrollen Sie zu dem Abschnitt &quot;Identifying data and researchers&quot;).</help>
```
Comment (Zack-83) [inline rdmorganiser/questions/rdmo.xml:3617]:
```suggestion

Weitere Informationen zur Funktionsweise, Verwendung und den verschiedenen Arten von PIDs findet man z. B. im &lt;a href=&quot;https://www.dpconline.org/handbook/technical-solutions-and-tools/persistent-identifiers&quot; target=&quot;_blank&quot;&gt; Handbook der Digital Preservation Coalition&lt;/a&gt; oder in den Informationsmaterialien des &lt;a href=&quot;http://www.ands.org.au/guides#identify&quot; target=&quot;_blank&quot;&gt; Australian National Data Service (ANDS)&lt;/a&gt; (bitte scrollen Sie zu dem Abschnitt &quot;Identifying data and researchers&quot;).</help>
```

**Comment by Zack-83** [comment]
Bei den Markdown-Listen muss eine doppelte Neuzeile sein oder ist sie überflüssig? In beiden Fällen ist die Arbeit nicht ganz fertig, sehe ich gerade

grafik

**Review by torsten-rathmann — APPROVED** [review]

**Comment by torsten-rathmann** [comment]
These unnecessary newlines and spaces are not serious, nevertheless they are disturbing when editing the questionnaire. Thanks to 
Giacomo for adjusting. The new file can be imported into RDMO 1.6.1 without problems.

**Comment by jmfrenzel** [comment]
> Bei den Markdown-Listen muss eine doppelte Neuzeile sein oder ist sie überflüssig? In beiden Fällen ist die Arbeit nicht ganz fertig, sehe ich gerade
> 
> grafik

@Zack-83 There is no obvious need for a separating new line between individual items in a list. Rdmo will render and display this as expected. 
**However**, when it comes to exporting a list from  rdmo, which does not have the items separated by newlines you will get 

Import:
```
<help lang="en">Examples of discipline-specific requirements are: 

- <a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/guidelines_biodiversity_research.pdf" target="_blank">Guidelines on the Handling of Research Data in Biodiversity Research</a>

- <a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/richtlinien_forschungsdaten_bildungsforschung.pdf" target="_blank">Guidelines for the provision and use of quantitative data in education research (German)</a>

- <a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/foerderkriterien_editionen_literaturwissenschaft.pdf" target="_blank">Elegibility criteria for 
funding for scholarly editions in the literary studies (German)</a>

- Recommendations on <a href="http://www.dfg.de/download/pdf/foerderung/grundlagen_dfg_foerderung/informationen_fachwissenschaften/geisteswissenschaften/standards_sprachkorpora.pdf" target="_blank">data standards and tools</a> as well as <a href="http://www.dfg.de/download/pdf/foerderung/grundlagen_dfg_foerderung /informationen_fachwissenschaften/geisteswissenschaften/standards_recht.pdf" target="_blank">legal aspects</a>
</help>
```
RDMO export:
```
<help lang="en">
Examples of discipline-specific requirements are: - <a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/guidelines_biodiversity_research.pdf" target="_blank">Guidelines on the Handling of Research Data in Biodiversity Research</a> - <a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/richtlinien_forschungsdaten_bildungsforschung.pdf" target="_blank">Guidelines for the provision and use of quantitative data in education research (German)</a> - <a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/foerderkriterien_editionen_literaturwissenschaft.pdf" target="_blank">Elegibility criteria for funding for scholarly editions in the literary studies (German)</a> - Recommendations on <a href="http://www.dfg.de/download/pdf/foerderung/grundlagen_dfg_foerderung/informationen_fachwissenschaften/geisteswissenschaften/standards_sprachkorpora.pdf" target="_blank">data standards and tools</a> as well as <a href="http://www.dfg.de/download/pdf/foerderung/grundlagen_dfg_foerderung/informationen_fachwissenschaften/geisteswissenschaften/standards_recht.pdf" target="_blank">legal aspects</a> associated with language corpora (German)
</help>
```

**Comment by jmfrenzel** [comment]
@Zack-83 I have to partially correct myself :arrow_up: 
I have checked with rdmo v1.4 and rdmo v1.6 and it will always export one liners :disappointed:

**Comment by jmfrenzel** [comment]
> @Zack-83 I have to partially correct myself arrow_up I have checked with rdmo v1.4 and rdmo v1.6 and it will always export one liners disappointed

I meant that it is disappointing for those who are maintaining the content directly in xml. 

On re-import these onliners are displayed as expected, see  https://github.com/rdmorganiser/rdmo/issues/117#issuecomment-473287476

**Comment by jmfrenzel** [comment]
@Zack-83  Sorry for my confusing comments. When checking the exported xml-files I was using an xml viewer with pre formatting the content (one line per xml entry). 

The structure of the help text in the xml-file on disk (as it is processed by git) is preserved on export as pointed out by  https://github.com/rdmorganiser/rdmo/issues/117#issuecomment-473287476

So there is no issue for maintaining these files and therefor I see no reason which would items in lists to be separated by newlines, but they don't hurt.

**Comment by jmfrenzel** [comment]
@Zack-83 what is your plan with this PR
i)  do you want to go over the xml files and remove newlines that separate list items or 
ii) should we proceed as is

**Comment by Zack-83** [comment]
> 
> 
> @Zack-83 what is your plan with this PR i) do you want to go over the xml files and remove newlines that separate list items or ii) should we proceed as is

@jmfrenzel I will remove the double newlines. If it helps, I can also end the previous line with a double space

**Review by jmfrenzel — APPROVED** [review]
Looks good.

**Pull request merged**